jdaviz/configs/cubeviz/plugins/sonify_data/tests/test_sonify_data.py

```diff
@@ -29,7 +29,8 @@ def test_sonify_data(cubeviz_helper, spectrum1d_cube_larger):
     subset_plugin.import_region(spec_region)
     sonify_plg.spectral_subset_selected = 'Subset 1'
     sonify_plg.vue_sonify_cube()
-    assert sonify_plg.flux_viewer.sonification_wl_bounds == (4.62360028e-07, 4.62920561e-07)
+    assert sonify_plg.flux_viewer.sonification_wl_ranges[0] == (4.62360028e-07 * u.m,
```

Comment:
Why would the glue change touch this?

Reply:
This is a test that is only run locally and the attribute had been changed from `sonification_wl_bounds` to `sonification_wl_ranges` in #3377. I just decided to update it since I was running tests locally and seeing that failure.